boards/inga_common/include/board_common.h

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2016 Robert Hartung <hartung@ibr.cs.tu-bs.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+#ifndef INGA_COMMON_BOARD_COMMON_H_
+#define INGA_COMMON_BOARD_COMMON_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define UART_STDIO_BAUDRATE               (19200)
+
+/**
+ ** Context swap defines
+ ** Setup to use PA0 which is pin change interrupt 0 (PCINT0)
+ ** This emulates a software triggered interrupt
+ ***/
+#define AVR_CONTEXT_SWAP_INIT do { \
+            DDRA |= (1 << DDA0); \
+            PCICR |= (1 << PCIE0); \
+            PCMSK0 |= (1 << PCINT0); \
+} while (0)
+#define AVR_CONTEXT_SWAP_INTERRUPT_VECT         PCINT0_vect
+#define AVR_CONTEXT_SWAP_INTERRUPT_VECT_NUM     PCINT0_vect_num
+#define AVR_CONTEXT_SWAP_TRIGGER                PORTA ^= (1 << PA0)
+
+/**
+ * Context swap defines
+ * Setup to use WDT interrupt
+ */
+/*
```

Comment:
Are these left-overs? If not please, add as doxygen when documenting `AVR_CONTEXT_SWAPT_INIT`

Reply:
It's an alternative implementation that uses the WDT interrupt, in case someone needs the port interrupt pin.

Reply:
Mmm I've no strong opinion here... 
Another thing that I just realized about (sorry :-P): Why do you need the `tmp` variable? Does the interrupt trigger after enabling it? And yet another...: you set twice the `WDIE` flag twice in tmp.

```C
MCUSR &= ~(1 << WDRF);\
WDTCSR = (1 << WDCE) | (1 << WDIE) | (1 << WDE);\
WDTCSR = (1 << WDCE) | (1 << WDIE) | (1 << WDE);\ # this isthe second tmp assignment
```

Reply:
I think the WDT should not be used this way. There's a plan to provide a WDT with a RIOT API so for now I'd say that removing this code is cleaner.